src/containers/Tables/BidTable/BidTableRow.tsx

```diff
@@ -0,0 +1,70 @@
+import styles from './BidTable.module.scss';
+
+import { Artwork } from 'components';
+import { ethers } from 'ethers';
+import { CancelBidButton } from 'containers';
+
+export type BidTableRowData = {
+	domainName: string;
+	auctionId: string;
+	domainId: string;
+	domainMetadataUrl: string;
+	date: Date;
+	yourBid: ethers.BigNumber;
+	highestBid: ethers.BigNumber;
+};
+
+export const TEST_ID = {
+	CONTAINER: 'bid-table-row-container',
+	ARTWORK: 'bid-table-row-artwork',
+	HIGHEST_BID: 'bid-table-row-highest-bid',
+	YOUR_BID: 'bid-table-row-your-bid',
+	DATE: 'bid-table-row-date',
+	STATUS: 'bid-table-row-status',
+};
+
+const BidTableRow = (props: any) => {
+	const bid: BidTableRowData = props.data;
+	const isHighestBid = bid.yourBid.gte(bid.highestBid);
+
+	const onSuccess = () => {
+		props.onRefetch();
+	};
+
+	return (
+		<tr className={styles.Container} data-testid={TEST_ID.CONTAINER}>
+			<td>
+				<Artwork
+					data-testid={TEST_ID.ARTWORK}
+					domain={bid.domainName}
+					disableInteraction
+					metadataUrl={bid.domainMetadataUrl}
+					id={bid.domainId}
+					style={{ maxWidth: 200 }}
+				/>
+			</td>
+			<td data-testid={TEST_ID.DATE}>{bid.date.toLocaleDateString()}</td>
+			<td data-testid={TEST_ID.YOUR_BID}>
+				{ethers.utils.formatEther(bid.yourBid.toString())} WILD
+			</td>
+			<td data-testid={TEST_ID.HIGHEST_BID}>
+				{ethers.utils.formatEther(bid.highestBid)} WILD
+			</td>
+			<td
+				data-testid={TEST_ID.STATUS}
+				className={isHighestBid ? styles.Lead : styles.Outbid}
+			>
+				{isHighestBid ? 'You Lead' : 'Outbid'}
```

Comment:
nit: It will be better to have this text as a constant and use it in both of this component and above test files.

```
const TEXTS = {
   STATUS: {
     NORMAL: 'Outbid',
    HIGHLIGHT: 'You Lead'
   }
};
```

So we can change the text in once place( here) instead of changing in both of component and test. (when we need to change the text)

Reply:
Good call. I also changed the `"WILD"` string to a constant in case we need to show a different token name at some point